### PR TITLE
Format endpoint properties - Fix #2199

### LINKED ISSUE #2199: Include details regarding 'legacy-encoding' in Micro Integrator documentation
**Description:**
It would be better if details regarding the use of `legacy-encoding:` while configuring HTTP endpoint parameters are included in the Micro Integrator documentation. The details are already available in the Enterprise Integrator documentation [1].

**Affected Product Version:**
EI 7.1.0

[1] https://docs.wso2.com/display/EI650/HTTP+Endpoint#HTTPEndpoint-Parameters

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
Fixes #2199 

**Related PRs**

#2598 
#2597

### CODE REVIEW

**Review by Nirothipan — APPROVED** [review]

File: en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md [hunk 1]
```diff
@@ -412,16 +412,19 @@ Member properties can be used to associate configuration data with an endpoint.
...
-                        <pre class="sourceCode java">
-                            <code class="sourceCode java"><span id="cb1-1"><a href="#cb1-1"></a>&lt;http uri-template=<span class="st">&quot;http://{uri.var.host}:{uri.var.port}/{uri.var.context1}/services/{uri.var.context2}{uri.var.postfix}&quot;</span> method=<span class="st">&quot;get&quot;</span>&gt;</span></code>
-                        </pre>
-                     </div>
+            The URI template that constructs the RESTful endpoint URL at runtime. Insert <code>uri.var.</code>  before each variable.</br>
+            <b>Notes</b>:
+            <ul>
+               <li>If the endpoint URL is an encoded URL, then you need to add <code>legacy-encoding:</code> when defining the uri-template. For example, <code>uri-template="legacy-encoding:{uri.var.APIurl}"</code>.</br></li>
+               <li>If you add a <code>/</code> between <code>{uri.var.context2}{uri.var.postfix}</code>, it is evaluated as an invalid invocation because <code>{uri.var.postfix}</code> contains a <code>/</code> at the start.</li>
```
Comment (Nirothipan) [inline en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md:419]: let's update this as discussed offline

Comment (SanojPunchihewa) [reply en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md]: Fixed with 194a437

**Comment by samgnaniah** [comment]
Was this tested locally? Tables don't always render properly so just checking.

File: en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md [hunk 2]
```diff
@@ -412,19 +412,34 @@ Member properties can be used to associate configuration data with an endpoint.
...
                </div>
-            </div>
+               </li>
+               <li>If you add a <code>/</code> between <code>{uri.var.context2}{uri.var.postfix}</code>, it is evaluated as an invalid invocation because <code>{uri.var.postfix}</code> contains a <code>/</code> at the start.
+               <p>Example:</p>
+               <div class="code panel pdl" style="border-width: 1px;">
+                  <div class="codeContent panelContent pdl">
+                     <div class="sourceCode" id="cb1" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence">
+                        <pre class="sourceCode java">
+                              <code class="sourceCode java"><span id="cb1-1"><a href="#cb1-1"></a>&lt;http uri-template=<span class="st">&quot;http://{uri.var.host}:{uri.var.port}/{uri.var.context1}/services/{uri.var.context2}{uri.var.postfix}&quot;</span> method=<span class="st">&quot;get&quot;</span>&gt;</span></code>
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md:436]: Does this render correctly when you build locally?

Comment (SanojPunchihewa) [reply en/micro-integrator/docs/references/synapse-properties/endpoint-properties.md:436]: yeah
Screenshot from 2020-12-22 14-48-52

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**